server/src/main/java/org/apache/iotdb/db/client/ConfigNodeClient.java

```diff
@@ -169,10 +168,17 @@ public TDataNodeRegisterResp registerDataNode(TDataNodeRegisterReq req)
     for (int i = 0; i < RETRY_NUM; i++) {
       try {
         TDataNodeRegisterResp resp = client.registerDataNode(req);
-        if (!updateConfigNodeLeader(resp.status)) {
+
+        // check if config nodes are inconsistent
```

Comment:
describe more detail, confignode_list fetched from ConfigNode are inconsistent with which?

Reply:
I change the process here... When it needs to redirect to config node leader, set latest config node list in client to reconnect.